webapp/channels/src/components/admin_console/group_settings/groups_list/index.ts

```diff
@@ -4,12 +4,11 @@
 import {connect} from 'react-redux';
 import {bindActionCreators, Dispatch, ActionCreatorsMapObject} from 'redux';
 
-import {createSelector} from 'reselect';
+import {GlobalState} from '@mattermost/types/store';
```

Comment:
Why moving this import?

Reply:
Similar with other files with apparently unrelated import reordering

Reply:
I should genuinely have eslint errors for these import sorting. Something @hmhealey and I thought we would do before but never got to it.